src/views/DefineCompany.vue

```diff
@@ -2,7 +2,9 @@
   <div>
     <div class="mt-10 company-statement">
       <p>
-        <span class="company-statement-label">{{ getCompanyResources.title }}:</span>
+        <span v-if="getCompanyResources.description" class="company-statement-label">
+          {{ getCompanyResources.title }}:
```

Comment:
Wait - if there's a description then display the title and the description? I'm confused maybe.

Reply:
:) I suppose either property could have been used here. The statement title or the description.
But you are right, in the even there is no description that we want to display (LTD's) there is no title either.

Reply:
Could you clarify the logic in the code pretty please?

Reply:
Absolutely, i think it makes sense to check the title opposed to the description as well.

Reply:
This was actually a snafu on my part. 

I had the statement title in the resource file where it wasn't required. 
So opposed to having a condition i just removed the statement title (because there is none for LTD) and moved the colon into the titles.
So no conditions required, just taken from the Resource file verbatim.